gio/src/subprocess_launcher.rs

```diff
@@ -19,40 +19,41 @@ impl SubprocessLauncher {
     #[cfg(unix)]
     #[cfg_attr(docsrs, doc(cfg(unix)))]
     #[doc(alias = "g_subprocess_launcher_take_fd")]
-    pub fn take_fd(&self, source_fd: impl IntoRawFd, target_fd: impl IntoRawFd) {
+    pub fn take_fd(&self, source_fd: Option<OwnedFd>, target_fd: Option<OwnedFd>) {
```

Comment:
Only `source_fd` is taken ownership of, `target_fd` is just borrowed> AFAIU

> This function takes ownership of the source_fd; it will be closed in the parent when self is freed.